### DISCUSSION TITLE
Fleed Standalone Node

### DISCUSSION THREAD

**Discussion by ByteAndBits** [Security-Onion-Solutions/securityonion#11236] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

64

### RAM

256GB

### Storage for /

Raid5 

### Storage for /nsm

Raid10

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hi,

we set up Security Onion with scaling option. At which point we need a seperate fleet node?
At the end of all we will have 400+ agents in different locations. We start with around 50 and when erverything worked we scale up.

Can we build a feet cluster or is only one node possible? Can we setup the standalone fleet node later?

We don´t find anything about fleet standalone node in doku (hardware, sizing, config,....)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by kvkamin** [comment]
I have deployed over 400 elastic windows agents with endpoints-initial Policy. CPU Load on MasterSearch has gone up dramatically.  With wazuh agents, server processor load was about 6-8 (linux load), now with elastic agent it hovers around 18.  I suspect that the Elastic Defend integrations is causing most of increased load, as well as client side memory use. And  Some/All/Cool features are turned off unless you have a Elastic Premium license.  After creating new Policy without Elastic Defend, pushing it to all windows systems, cpu load dropped to 8-9. Receive traffic on master interface dropped from ave 7.5 Mbps to around 4 Mbps.

**Reply by 8bitjoe** [reply]
good to know! thanks

**Comment by TOoSmOotH** [comment, marked as answer]
You can add fleet hosts at any time. Keep in mind that all fleet agents will attempt to connect to them as there is only a single elastic agent output policy. You have to use the firewall to control connectivity. @kvkamin Thanks for metrics!
